src/helpers/ui/filter_interactive.ts

```diff
@@ -17,3 +17,58 @@ export function interactiveAddFilter(env: SpreadsheetChildEnv, sheetId: UID, tar
     env.raiseError(AddFilterInteractiveContent.nonContinuousTargets);
   }
 }
+
+export function adjecentCellsSelection(
```

Comment:
```suggestion
export function adjacentCellsSelection(
```

Reply:
If you use VSCode, you can install this extension, it will help point up this kind of mistake: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker